src/components/SecondaryPanes/XHRBreakpoints.css

```diff
@@ -82,11 +82,36 @@
 }
 
 .xhr-input-method {
-  display: none;
+  visibility: hidden;
+  margin-inline-start: 4px;
+  padding-block: 0;
+  padding-inline-start: 2px;
+  padding-inline-end: 14px;
+  border: none;
+  border-radius: 2px;
+  font: inherit;
+  color: inherit;
+  -moz-appearance: none;
+  appearance: none;
+  background-color: var(--toolbarbutton-background);
+}
+
+.xhr-input-container.focused::after {
+  content: "";
+  position: relative;
+  right: 25px;
+  top: 5px;
```

Comment:
A more robust way to center an absolutely positioned element of known height (8px) would be: `top: calc(50% - 4px)`.

Reply:
Alrighty, done!